packages/ragbits-document-search/src/ragbits/document_search/documents/element.py

```diff
@@ -90,23 +90,35 @@ def from_vector_db_entry(cls, db_entry: VectorStoreEntry) -> "Element":
         """
         element_type = db_entry.metadata["element_type"]
         element_cls = Element._elements_registry[element_type]
+        if "embedding_type" in db_entry.metadata:
+            del db_entry.metadata["embedding_type"]
         return element_cls(**db_entry.metadata)
 
-    def to_vector_db_entry(self, vector: list[float]) -> VectorStoreEntry:
```

Comment:
why is `embedding_type` optional here?

Reply:
I think embedding type should be enforced here

Reply:
done